app/helpers/exchange_rates_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ExchangeRatesHelper
+  def filter_years(years, year_to_hide)
+    years.reject { |year| year.year == year_to_hide || year.year == ExchangeRates::Year::HIDDEN_YEAR }
```

Comment:
just to ask, is the year a year? 😄  or is it a date class?
`year.year` sounds a bit strange

Reply:
I think filter_years makes more sense because we provide a year to filter out and we also remove years we don't want